### PR TITLE
Ambiguity measure

### PR DESCRIPTION
**Pull request by jelledouwe**
This PR contains the AmbiguityMeasure tool for human robot interaction. It is related to one of our recent publications. We also created a [colab](https://colab.research.google.com/github/opendr-eu/opendr/blob/ambiguity_measure/projects/python/utils/ambiguity_measure/ambiguity_measure_tutorial.ipynb).

### CODE REVIEW

File: docs/reference/ambiguity_measure.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+The [AmbiguityMeasure](../../src/opendr/utils/ambiguity_measure/ambiguity_measure.py) class has the following public methods:
+
+#### `AmbiguityMeasure` constructor
+```python
+AmbiguityMeasure(self, threshold, temperature)
+```
+
+Constructor parameters:
+
+- **threshold**: *float*\
```
Comment (ad-daniel) [inline docs/reference/ambiguity_measure.md:20]:
```suggestion
- **threshold**: *float, default=0.5*\
```

File: docs/reference/ambiguity_measure.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  Ambiguity threshold, should be in [0, 1).
+- **temperature**: *float*\
```
Comment (ad-daniel) [inline docs/reference/ambiguity_measure.md:22]:
```suggestion
- **temperature**: *float, default=1.0*\
```

File: docs/reference/ambiguity_measure.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  Temperature of the sigmoid function. Should be > 0.
```
Comment (ad-daniel) [inline docs/reference/ambiguity_measure.md:23]:
```suggestion
  Temperature of the sigmoid function.
  Should be > 0.
```

File: docs/reference/ambiguity_measure.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  These can be obtained using from for example a Transporter Nets model [[1]](#transporter-paper).
+
+#### Demos and tutorial
+
+A demo showcasing the usage and functionality of the *AmbiguityMeasure* is available [here](https://colab.research.google.com/github/opendr-eu/opendr/blob/ambiguity_measure/projects/python/utils/ambiguity_measure/ambiguity_measure_tutorial.ipynb).
+
+
+#### Examples
+
+* **Ambiguity measure example.**
```
Comment (ad-daniel) [inline docs/reference/ambiguity_measure.md:46]:
```suggestion
* **Ambiguity measure example**
```

File: docs/reference/ambiguity_measure.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  ```
+
+#### References
+<a name="transporter-paper" href="https://proceedings.mlr.press/v155/zeng21a/zeng21a.pdf">[1]</a>
+Zeng, A., Florence, P., Tompson, J., Welker, S., Chien, J., Attarian, M., ... & Lee, J. (2021, October).
+Transporter networks: Rearranging the visual world for robotic manipulation.
+In Conference on Robot Learning (pp. 726-747).
+PMLR.
+
+
```
Comment (ad-daniel) [inline docs/reference/ambiguity_measure.md:76]:
```suggestion
```

File: src/opendr/utils/ambiguity_measure/__init__.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+from opendr.utils.ambiguity_measure.ambiguity_measure import AmbiguityMeasure
+
+
```
Comment (ad-daniel) [inline src/opendr/utils/ambiguity_measure/__init__.py:3]:
```suggestion
```

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Thank you, being an utility I think what you did is sufficient. I have however a couple of comments:

- you have created a test (and locally it worked for me) but currently the tool is not being tested in the CI (you need to add it to the .yml files, both master and develop)
- I believe the the intent is for this tool to be distributed separately (like we do for the hyperparameter tuner), in which case you also need to add it to packages.txt so that one is created for it
- add a changelog entry

**Comment by jelledouwe** [comment]
> Thank you, being an utility I think what you did is sufficient. I have however a couple of comments:
> 
>     * you have created a test (and locally it worked for me) but currently the tool is not being tested in the CI (you need to add it to the .yml files, both master and develop)
> 
>     * I believe the the intent is for this tool to be distributed separately (like we do for the hyperparameter tuner), in which case you also need to add it to packages.txt so that one is created for it
> 
>     * add a changelog entry

Hi Daniel, thanks for the review! I believe the test is currently already tested in the CI. At least, it appears in the actions on this branch under the util tests. I added the entry to the package list and the changelog.

**Review by ad-daniel — DISMISSED** [review]
Yes you're right, it's being run in the `utils` job, I didn't see it. Thank you!

**Review by tsampazk — DISMISSED** [review]
Changes look good to me, thank you!

**Review by ad-daniel — APPROVED** [review]

**Review by tsampazk — APPROVED** [review]

**Pull request merged**